Tools/DeployApps/BaasClient.cs

```diff
@@ -41,8 +41,19 @@ public static class AppConfigType
         public const string FlexibleSync = "flx";
     }
 
-    public class BaasClient : IDisposable
+    public class BaasClient
     {
+        public class FunctionReturn
+        {
+            public enum Result
+            {
+                success = 0,
```

Comment:
These should be documented or the warning should be suppressed.

Reply:
I missed it. I'm gonna suppress it. It's only for our internal functions results.

Reply:
You can see the warning when you browse the code:

<img width="1357" alt="Screen Shot 2022-07-01 at 18 23 51" src="https://user-images.githubusercontent.com/2315687/176923662-d7915f20-955b-4e60-87e4-08f32cbac7e2.png">

To see that in VS, you'll probably need to rebuild the project, otherwise msbuild will not show warnings for code it doesn't compile.